### PR TITLE
Added chart part I

### PR DESCRIPTION
**Pull request by LiMengyang990726**
In our use-case, added top-10-active users using BarChart. Shown as below:
image
Updates are elaborated in the comments below.

### CODE REVIEW

**Comment by nanda-thant-sin** [comment]
Most of them are related to book not travelling. I don't think we need to commit these things to our source.

**Comment by LiMengyang990726** [comment]
> Most of them are related to book not travelling. I don't think we need to commit these things to our source.

I have changed the data. There are three charts for now.
1. Top-10-active-users and the number of messages they sent. Due to the lack of data, I haven't implemented the Java Servlet, but this is for future usage.
2. Geochart which is supposed to show the frequencies that a place has been visited.
3. I want this to the rating of popular places read from csv. As the only current csv that I have is the book one, and I haven't mocked up dataset, this is for future usage.
Basically, they contain the logic that I am going to use for the future. However, as currently I don't have relavant data and haven't mocked up dataset, some of them may look irrelavant for now.

I will check with our PA if I am able to commit them.

**Comment by LiMengyang990726** [comment]
> I think I should approve first, so that the next pull requests will be smaller sizes.

Thank you! I will check with our PA to see how can I pass the stickler.
